base/data/gm4-1.1/functions/log_start.mcfunction

```diff
@@ -0,0 +1,10 @@
+scoreboard players set @a[gamemode=creative] gm4_creative 0
+scoreboard players add @a[gamemode=!creative] gm4_creative 1
+
+execute if entity @a[gamemode=creative] run scoreboard players set $global gm4_creative 0
+execute unless entity @a[gamemode=creative] run scoreboard players add $global gm4_creative 1
+
+tag @a remove gm4_show_log
+execute as @a if score @s gm4_creative < $cooldown gm4_creative run tag @s add gm4_show_log
+execute if score $global gm4_creative >= $global_cooldown gm4_creative run tag @a add gm4_show_log
```

Comment:
Is this tag supposed to go to all players or only creative players?

Reply:
To all players. That's the third point mentioned
> When no players in creative have been seen for 10 reloads, it again starts showing logs to all players. This number is configurable.